chromadb/api/__init__.py

```diff
@@ -257,7 +257,7 @@ def _delete(
         ids: Optional[IDs],
         where: Optional[Where] = {},
         where_document: Optional[WhereDocument] = {},
-    ) -> List[UUID]:
+    ) -> IDs:
```

Comment:
I think the UUIDs are returned here not the IDs?

Reply:
**local**
https://github.com/chroma-core/chroma/blob/d88860bbc50f6b25eac568dbefee380440c3f27d/chromadb/api/local.py#L413

**clickhouse**
https://github.com/chroma-core/chroma/blob/d88860bbc50f6b25eac568dbefee380440c3f27d/chromadb/db/clickhouse.py#L514

**duckdb**
https://github.com/chroma-core/chroma/blob/d88860bbc50f6b25eac568dbefee380440c3f27d/chromadb/db/duckdb.py#L341